hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -335,7 +335,7 @@
     },
     { name: "spi_host0",
       type: "spi_host",
-      clock_srcs: {clk_i: "io_div4", clk_core_i: "io"},
+      clock_srcs: {clk_i: "io_div2", clk_core_i: "io"},
       clock_group: "peri",
       reset_connections: {rst_ni: "spi_host0", rst_core_ni: "spi_host0"},
```

Comment:
we probably need to create one more reset that is synchronous to `io` clock. Since the `core` domain of spi_host is actually operating at a much higher frequency asynchronous clock.  I can do this separately as well.

Reply:
yeah you are right. I have created a second reset node for each of the spi hosts.